hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -60,14 +61,39 @@ public HoodieCommitArchiveLog(HoodieWriteConfig config, FileSystem fs) {
      */
     public boolean archiveIfRequired() {
         List<HoodieInstant> commitsToArchive = getCommitsToArchive().collect(Collectors.toList());
```

Comment:
agreed. I am going to revamp archival in #165 and even the class name has to change. This is a fix to make sure we dont leave the .clean files around in COW.